src/main/java/playfriends/mc/plugin/playerdata/PlayerData.java

```diff
@@ -29,6 +29,14 @@ public class PlayerData {
     @Persistent("afk-enabled")
     private boolean isAfkEnabled = true;
 
+    /** The player's preferred pronouns. */
```

Comment:
nitpick: I'd drop the word 'preferred' here, since you're talking about someone's actual pronouns and not simply a preference 🙂

Reply:
Oh my.., yes of course, will change that

Must've recalled the phrase from somewhere without really thinking about it